pkg/controller/reconciler/main_test.go

```diff
@@ -34,6 +36,15 @@ var config *rest.Config
 // scheme holds a reference to the scheme for the test environment.
 var scheme *runtime.Scheme
 
+// webhookInstallOptions holds a reference to the webhook install options for the test environment.
+var webhookInstallOptions *envtest.WebhookInstallOptions
```

Comment:
It looks like this is only used inside `initializeWebhookInEnvironment`. Do you think we could get away without a global variable? Global variables are scary 👻

Reply:
The variable was needed in recipe_webhook_test.go https://github.com/radius-project/radius/blob/c117c5baa63df34a7b36491e69534d9d808dd467/pkg/controller/reconciler/recipe_webhook_test.go#L225 I tried a couple things but perhaps the only way to get rid of it completely is to have the mgr and webhook up and running in for all tests including reconciler tests which I don't think we want.  I limited the number of fields and updated naming of the variable to be obvious and overall am following similar pattern as the reconciler tests. Let me know if okay.

Reply:
If we can't avoid it, then we can't avoid it 😢